ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot

```diff
@@ -15,17 +15,17 @@ Is RHODS Installed
               ${result}=  Run Keyword And Return Status
               ...  Run Keywords
               ...  Check A RHODS Family Operator Is Installed  namespace=redhat-ods-operator
-              ...                                              subscription=rhods-operator-dev  AND
+              ...                                              subscription=rhoai-operator-dev  AND
```

Comment:
what's the migration path if I install with old ods-ci, then pull this change, and then want to run ods-ci again?

(I'm trying to find something to complain and I just got notified from this PR, so, found something to complain about)

Reply:
there is no migration path from my side :smile:

Reply:
For the deployment with old CS name can be uninstalled manually and from then we can start using this.